.travis.yml

```diff
@@ -52,11 +54,11 @@ before_install:
   - ./miniconda.sh -b -p $MINICONDA
   - export PATH=$MINICONDA/bin:$PATH
   - conda update --yes conda
-  - if [ -n "${FITS}" ]; then conda install --yes -c ${SHERPA_CHANNEL} ${FITS}; fi
+  - if [ -n "${FITS}" ]; then conda install --yes -c ${SHERPA_CHANNEL} ${FITS} $MKL; fi
   - if [ -n "${MATPLOTLIBVER}" ]; then MATPLOTLIB="matplotlib=${MATPLOTLIBVER}"; fi
   - if [ -n "${NUMPYVER}" ]; then NUMPY="numpy=${NUMPYVER}"; fi
   - echo ${MATPLOTLIB} ${NUMPY} ${FITS}
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pip ${MATPLOTLIB} ${NUMPY} nomkl
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION pip ${MATPLOTLIB} ${NUMPY} $MKL libgfortran=$LIBGFORTRANVER
```

Comment:
I thikn it would be safer - in the sense of much-less-likely to end up with a mis-matched set of libraries - if we could just have a single `conda install` line with all the requirements listed on it. This way conda can resolve all the dependencies cleanly. I have occasionally seen issues when updating/downgrading dependencies (in particular if they need different numpy ABIs). I think it would also have the advantage of making it a bit easier to trace what is/isn't being installed in a particular run.

The other approach to take would be to say that if a problem does exist we can quickly update the this file, and it's not worth the engineering time to make this "clean".